kube-prereq.adoc

```diff
@@ -8,28 +8,26 @@
 ////
 == Prerequisites
 
-Before you begin, have the following tools installed:
+Before you begin, the following tools must be installed:
 
 
-First, you need a containerization software for building containers. {kube} supports various container runtimes. You will use `Docker` in this guide. For installation instructions, refer to the https://docs.docker.com/install/[official Docker documentation^].
+First, you need a containerization software for building containers. {kube} supports various container runtimes. You will use `Docker` in this guide. For Docker installation instructions, refer to the https://docs.docker.com/install/[official Docker documentation^].
 
 ****
 [system]#*{win} | {mac}*#
 
-Use Docker Desktop, where a local {kube} environment is pre-installed and enabled. If you do not see the {kube} tab then you have an older version of Docker Desktop; upgrade to the latest version.
+Use Docker Desktop, where a local {kube} environment is pre-installed and enabled. If you do not see the _{kube}_ tab, then upgrade to the latest version of Docker Desktop.
 
 Complete the setup for your operating system:
 
- - Set up  https://docs.docker.com/docker-for-windows/#kubernetes[Docker for Windows^]. On the Docker for Windows _General Setting_ page, ensure that the option `Expose daemon on 		
-  tcp://localhost:2375 without TLS` is enabled. This is required by the `dockerfile-maven` 		
-  part of the build.
- - Set up https://docs.docker.com/docker-for-mac/#kubernetes[Docker for Mac^].
+  - Set up  https://docs.docker.com/docker-for-windows/#kubernetes[Docker for Windows^]. On the Docker Preferences, ensure that the `Expose daemon on tcp://localhost:2375 without TLS` option is enabled in the _General_ tab. This setting is required by the `dockerfile-maven` plugin when it's used in the POM file.
```

Comment:
Change
"This setting is required by the `dockerfile-maven` plugin when it's used in the POM file."
To
"When the `dockerfile-maven` plug-in is used in the POM file, this setting is required." OR "When this setting is used in the POM file, it is required by the `dockerfile-maven` plug-in." (Depending on which is correct.)

Reply:
The first one should be the right one if I am not mistaken.

Reply:
Yes, the first one is the right one

Reply:
Changed.